Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -849,8 +856,13 @@ protected static function ensureCLISubrequestsUseCurrentlyRunningPhpBinary($phpB
             return;
         }
 
-        exec(PHP_BINARY . ' -r "echo realpath(PHP_BINARY);"', $output);
-        $realPhpBinary = $output[0];
+        // stfu to avoid possible open_basedir restriction https://github.com/neos/flow-development-collection/pull/2491
+        $realPhpBinary = @realpath(PHP_BINARY);
```

Comment:
alternatively we can just check if `ini_get('open_basedir')` is not empty and then assume `realpath` will not work ...